src/main/osd/osd_elements.c

```diff
@@ -922,6 +922,17 @@ static void osdElementGpsSpeed(osdElementParms_t *element)
 {
     tfp_sprintf(element->buff, "%c%3d%c", SYM_SPEED, osdGetSpeedToSelectedUnit(gpsConfig()->gps_use_3d_speed ? gpsSol.speed3d : gpsSol.groundSpeed), osdGetSpeedToSelectedUnitSymbol());
 }
+
+static void osdElementEfficiency(osdElementParms_t *element)
+{
+    if (sensors(SENSOR_GPS) && ARMING_FLAG(ARMED) && STATE(GPS_FIX)) {
+        const int speed = osdGetSpeedToSelectedUnit(gpsSol.groundSpeed);
+        const int efficiency = speed > 0 ? getAmperage() * 10 / speed : 0;
```

Comment:
Instead of using cm/s, I will try multiply the speed by 100 to improve resolution at slow speeds.  This results in easier to understand code I think.

:+1: regarding `----` instead of `0` when speed is zero/low or efficiency is out of range.